### PR TITLE
Purchases can not be finished after restoreAllPurchases (Android)

### PR DESCRIPTION
**Pull request by jrouault**
Restored purchases have to be reset as ```PENDING``` so ```finishPurchase``` can be called again for each.

From now on, PR will be done on ```master``` instead of ```develop```. Release tags can be used to refer to a specific version.

poke @ronkorving

### CODE REVIEW

File: src/android/WizPurchase.java [hunk 1]
```diff
@@ -462,16 +462,23 @@ private void finishPurchase(JSONArray data) throws JSONException {
 	 * Get the list of purchases
 	 *
 	 **/
-	private JSONArray getPurchases() throws JSONException {
+	private JSONArray restorePurchasesAsPending() throws JSONException {
```
Comment (ronkorving) [inline src/android/WizPurchase.java]: I would add `All` in this name to indicate this is not about a selection (function argument) of purchases, but really about all purchases in the system.

Comment (ronkorving) [inline src/android/WizPurchase.java]: Alternatively, pass the `List<Purchase>` as an argument to thie function. That way you don't need to return it at the end, and you can simply call this function `setPurchasesAsPending(purchases)`

**Comment by jrouault** [comment]
Updated

**Comment by ronkorving** [comment]
lgtm

**Pull request merged**